documentation/src/main/asciidoc/stories/assembly_marshaller_implementations.adoc

```diff
@@ -0,0 +1,17 @@
+[id='marshaller_implementations']
```

Comment:
Is this story included anywhere?

Reply:
yep. it's nested under the main `assembly_marshalling.adoc` story.